### PR TITLE
New Package : typos fixed for French language

### PR DESCRIPTION
**Pull request by Jniceid**
Collection of typos fixed for French language

### CODE REVIEW

**Comment by AucaCoyan** [comment]
No response, closing

File: packages/French-Corrector/package.yml [hunk 1]
```diff
@@ -0,0 +1,7899 @@
...
+- trigger: numero
+  replace: numéro
+  propagate_case: true
+  word: true
+- trigger: pratique
+  replace: pratique
+  propagate_case: true
+  word: true
+- trigger: public
+  replace: public
```
Comment (smeech) [inline packages/French-Corrector/package.yml:25]: And these?
And several others below.

**Review by smeech — COMMENTED** [review]
A few `- trigger:`/`replace:` pairs where the two words are the same.

File: packages/French-Corrector/package.yml [hunk 2]
```diff
@@ -0,0 +1,7899 @@
...
+- trigger: universite
+  replace: université
+  propagate_case: true
+  word: true
+- trigger: numero
+  replace: numéro
+  propagate_case: true
+  word: true
+- trigger: pratique
+  replace: pratique
```
[the remark this answers is not in the document]
Comment (AucaCoyan) [reply packages/French-Corrector/package.yml:21]: Yup, we can add a lint to check if we have two colliding triggers, that would be nice

[the remark this answers is not in the document]
Comment (smeech) [reply packages/French-Corrector/package.yml:21]: > Yup, we can add a lint to check if we have two colliding triggers, that would be nice

Well, it's not colliding triggers so much as trigger/replace pairs that don't actually change anything.

**Pull request closed without merging**